themes/bootstrap3/templates/myresearch/mylist.phtml

```diff
@@ -28,9 +28,9 @@
 <?=$this->component('show-account-menu-button')?>
 
 <div class="<?=$this->layoutClass('mainbody')?>">
-  <h2><?=$list ? $this->escapeHtml($list->title) : $this->transEsc('saved_items')?></h2>
+  <h2><?=$list ? $this->escapeHtml($list->getTitle()) : $this->transEsc('saved_items')?></h2>
 
-  <?php if ($list && $user && $list->user_id === $user->id && $list->public): ?>
+  <?php if ($list && $user && $list->getUser()->getId() === $user->id && $list->isPublic()): ?>
```

Comment:
We could also change `$user->id` to `$user->getId()` here while we're updating things anyway.